### PR TITLE
[Optimize](inverted index) optimize inverted index bitmap copy

### PR DESCRIPTION
**Pull request by airborne12**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by airborne12** [comment]
run buildall

**Comment by airborne12** [comment]
run buildall

File: be/src/olap/rowset/segment_v2/inverted_index_reader.h [hunk 1]
```diff
@@ -249,15 +253,16 @@ class BkdIndexReader : public InvertedIndexReader {
...
+                 std::shared_ptr<roaring::Roaring>& bit_map) override;
     Status try_query(OlapReaderStatistics* stats, const std::string& column_name,
                      const void* query_value, InvertedIndexQueryType query_type,
                      uint32_t* count) override;
     Status invoke_bkd_try_query(const void* query_value, InvertedIndexQueryType query_type,
                                 std::shared_ptr<lucene::util::bkd::bkd_reader> r, uint32_t* count);
     Status invoke_bkd_query(const void* query_value, InvertedIndexQueryType query_type,
                             std::shared_ptr<lucene::util::bkd::bkd_reader> r,
-                            roaring::Roaring* bit_map);
+                            std::shared_ptr<roaring::Roaring> bit_map);
```
Comment (xiaokang) [inline be/src/olap/rowset/segment_v2/inverted_index_reader.h:265]: Why not & as other functions?

**Comment by airborne12** [comment]
run buildall

**Review by qidaye — APPROVED** [review]
LGTM

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**